### ISSUE TITLE
TOP PRIORITY: Alt text not working

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/LGBTQ-of-the-day-bot#105]
**Describe the bug**
Tweet alt text just shows `alt="Image"`. Here's [an example tweet](https://twitter.com/LgbtQotd/status/1317118324124299264). It should read the same text as what is being put on the image.

Alt text was added via PR #100 from issue #98 

**Expected behavior**
Alt text should read the same text as what is being put on the image.

**Screenshots**
![Screen Shot showing image not having actual alt text](https://user-images.githubusercontent.com/46823403/96292769-b7157280-0fb7-11eb-8c55-15aff98d61b0.png)

**Additional context**
Alt-text is currently at https://github.com/VickiLanger/LGBTQ-of-the-day-bot/blob/production/tweet.py#L50
`tweet_text` is currently at https://github.com/VickiLanger/LGBTQ-of-the-day-bot/blob/production/get_tweet_content.py#L16

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Comment by JasirZaeem** [comment]
Just noticed this as well, I am sending a pull request to hopefully fix this.

**Comment by VickiLanger** [comment]
Awesome, thanks! I really want to make sure the tweets are accessible to everyone. It's not quite the end of the world since the text is in the tweet, but still should be there. 

I'm still looking at the font issue. (and to be honest, having a heck of a time focusing today.)

**Comment by JasirZaeem** [comment]
I would like there to be alt text as well I think it is very important. Couldn't test it myself since I do not have twitter api keys, I have created a pull request to hopefully fix alt text, and a small error in replies.

**Comment by JasirZaeem** [comment]
That did not seem to have fixed it, I'll research it a bit to see if I can find a fix, I think this should remain open in case someone else is able to fix this

**Comment by VickiLanger** [comment]
Here's an example of working alt text. 
`api.create_media_metadata(media_id=gif_upload.media_id, alt_text="randomly generated gif, hopefully depicting a hug")`
[reference](https://github.com/achasveachas/hug-bot/blob/61f410e093fd12731968da87bc231a00b3d62665/hug-bot.py#L47)

**Comment by JasirZaeem** [comment]
[This](https://github.com/JasirZaeem/LGBTQ-of-the-day-bot/blob/82110d0e84579c680eb26c9c0584069757856491/tweet.py#L51) was the first thing I tried but it did not work unfortunately, I then tried passing the id in string form which is what the [current code does](https://github.com/VickiLanger/LGBTQ-of-the-day-bot/blob/production/tweet.py#L50).
I did not think it would make difference but maybe explicitly making it a named parameter like `media_id=image.media_id` would make it work, I'll make a pull request doing that.

**Comment by VickiLanger** [comment]
I'll check this out later.

current error:
```
An error occurred while generating image. reason: 'API' object has no attribute 'create_media_metadata'
```

But the tweep docs have this http://docs.tweepy.org/en/latest/api.html#API.create_media_metadata

**Comment by JasirZaeem** [comment]
I hadn't noticed the tweepy version in requirements.txt was not the latest one, create_media_metadata was added in the newest version which is likely why you get this error, I have created a PR to hopefully fix this #277

**Comment by VickiLanger** [comment]
Oh! You're right. That might be the problem. Good catch

**Comment by JasirZaeem** [comment]
![image](https://user-images.githubusercontent.com/20666236/99417035-7ba3f600-291f-11eb-998c-c43dc8dc15d4.png)
Alt text seems to be working now, I think this issue can be closed then

**Comment by VickiLanger** [comment]
I just saw that!!! Thanks again for figuring out what was wrong.

**Comment by JasirZaeem** [comment]
Just glad it's finally fixed after so long :D, finally got time to think about it after exams and other work
